articles/azure-large-instances/workloads/epic/find-your-subscription-id.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Find your subscription ID
+description: Explains how to find your ALI for Epic subscription ID.
+ms.topic: conceptual
+author: jjaygbay1
+ms.author: jacobjaygbay
+ms.service: baremetal-infrastructure
+ms.date: 06/01/2023
+---
+
+# Find your subscription ID
+
+A *Subscription ID* is a unique identifier for your service in Azure. 
+You will need it when interacting with the Microsoft Support team. 
+To find your subscription ID, follow these steps:
+
+1. Go to Azure support portal https://portal.Azure.Com
+2. From the left pane, select **Subscriptions**.
```

Comment:
@ianmsgit : Any reason why the images from Welcome package for this section are not ported?
Cc: @akashdubey-ms

Reply:
@suzizuber : Adding you here for you to address this comment.